### ISSUE TITLE
[DataCap Application] <Fengyundu> 

### ISSUE THREAD

**Issue by fengyunduHJKG** [filecoin-project/filecoin-plus-large-datasets#490]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: Beijing Fengyundu Technology Co.,Ltd.
- Website / Social Media:http://www.rekoo.co.jp/、http://gameapp.qq.com/605、http://gameapp.qq.com/25227
- Total amount of DataCap being requested (between 500 TiB and 5 PiB): 5PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB): 100TiB
- On-chain address for first allocation: f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
Beijing Fengyundu Technology Co.,Ltd. was established in 2013.We focus on the development and operation of mobile Internet applications.
As the first Chinese game company to achieve great success in overseas social platforms, its products are spread all over the world, covering high-quality social platforms in developed countries such as Japan, the United States, South Korea, Germany and Russia, and it ranks first among social game companies in China and Japan. 
In China, social game products are launched on social platforms such as Tencent, Weibo, and Renren, with hundreds of millions of active users.With the product concept of "boutique social games", the company will launch a variety of diversified social games based on PC and smart phones in the future to fully meet the growing diversified interactive entertainment needs of online gamers in the global market.
```

What is the primary source of funding for this project?
```
Company revenues.
```

What other projects/ecosystem stakeholders is this project associated with?
```
There is no other projects/ecosystem stakeholders associated with.
```

## Use-case details

Describe the data being stored onto Filecoin
```
The company's products have accumulated about 600 million registered users' game data for nearly ten years, as well as various game data: pictures, videos, database, texture, transactions, shader, audio,etc.
```

Where was the data in this dataset sourced from?
```
From the game, Mainly from users’ activities on the website.
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
http://gameview.fengyundu.com/fishing1.png
http://gameview.fengyundu.com/fishing2.png
http://gameview.fengyundu.com/fishing3.png
http://gameview.fengyundu.com/fishing4.png
http://gameview.fengyundu.com/fishing5.png
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
Yes, We confirm that the data we stored are all open source. 
```

What is the expected retrieval frequency for this data?
```
There is no fixed retrieval frequency for this data.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
3 years at least.
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
Greater China
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
Because the amount of data is too large, We will choose online transmission.
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
We are prefer to find suitable miners in the storage provider interface released by Filecoin.≤ 25% of our data will be sent to them.
```

How will you be distributing deals across storage providers?
```
We want to use DataCap and S3 bucket to store a portion of the data separately.
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
Sure! We have enough funds to support this project. 
```

**Comment by Sunnyiscoming** [comment]
@fengyunduHJKG 
[http://www.rekoo.co.jp](http://www.rekoo.co.jp/%E3%80%81http://gameapp.qq.com/605%E3%80%81http://gameapp.qq.com/25227) is operated by Rekoo Japan Co., Ltd
The official website of Beijing Fengyundu Technology Co.,Ltd.(北京风云度科技有限公司) is http://www.fengyundu.com/
Can you figure out the relationship of 北京风云度科技有限公司 with Rekoo Japan Co., Ltd, http://gameapp.qq.com/605, http://gameapp.qq.com/25227?

**Comment by fengyunduHJKG** [comment]
Hi，@Sunnyiscoming,
Rekoo is a brand.
Rekoo Japan Co., Ltd. and Beijing Fengyundu Technology Co.,Ltd.are two companies under the same brand.
Parts of the data for both companies are the same. such as user information.
1. fengyundu.rekoo.co.jp -> fengyundu.com
    rekoo.fengyundu.com  ->www.rekoo.co.jp
    We have set up their respective second-level domain names to point to the corresponding official website of the other party, and we have the right to manage the domain name.
2. The legal representative of both companies is the same person.
    https://gameview.fengyundu.com/add_pic1.png
    https://gameview.fengyundu.com/add_pic2.png
    https://gameview.fengyundu.com/add_pic3.png

**Comment by fengyunduHJKG** [comment]
Hi，@Sunnyiscoming,
The email has been sent, please check, thank you！

**Comment by raghavrmadya** [comment]
@fengyunduHJKG I would like more clarity on your DataCap allocation plan. We require geographic diversity and not all Data can be stored with SP's in China. Moving 5 PiBs of data online is no small feat. Please provide as much information as possible on SP distribution and how you plan to move the data to SPs.

**Comment by fengyunduHJKG** [comment]
@raghavrmadya 

We plan to store more than 180 days of archived data on this DataCap.

Data includes:
1 User's database backup, including full daily mysql, redis, etc.
2 User behavior analysis for bi.
3 Access_log, error_log, etc. of the server.
4. The user's weekly resource data, including avatars, chat records, etc.

Transmission:
We have a 100M private line from China to Japan that can be used (expanded according to demand in the future). At the same time, the S3 bucket data in Japan can also be directly transmitted.

**Comment by Sunnyiscoming** [comment]
Please provide as much information as possible on SP distribution

**Comment by raghavrmadya** [comment]
@fengyunduHJKG thanks for the response. Just to clarify, by SP distribution, we mean with which SP's you'll be storing the data and where are they located. Please provide their on-chain ID

**Comment by fengyunduHJKG** [comment]
@raghavrmadya 
At present, the general distribution of our data is:
China 35%, Japan 50%, South Korea 10%, USA 5%

We use public cloud storage (Tencent Cloud, Aliyun Cloud, AWS S3), and local server storage.
![1](https://user-images.githubusercontent.com/109143876/184060980-e5623ff0-00c4-4962-b82e-2d571eb548e4.png)

Our storage requirements: 
All data needs to be stored for 3-5 years for future listing audits. If using DataCap can reduce our costs, we will be happy to switch data storage.

1. Daily data backup (database, redis, other db), about 50 games, 1500 different types of databases, need more than 2T-8T space. (According to different backup cycles, there are daily, 3 days, 7 days to save costs)
2. BI data model, which currently stores more than 10 years of metadata (meta data), and more than 1000 data templates. Data archiving requires 50T per month, mainly metadata and analysis results.
3. access_log, error_log, 500G space is required per day after compression.
4. The advertising recommendation system based on user behavior will generate 200G-300g of compressed data every day.

In summary,about 2.5P-3.8P data storage is required per year, and the demand growth rate will be 30%-40% every year.

**Comment by raghavrmadya** [comment]
Thanks for the detailed response. Much appreciated.

**Comment by raghavrmadya** [comment]
### Datacap Request Trigger
**Total DataCap requested**

> 5 PiB

**Expected weekly DataCap usage rate**

> 100 TiB

**Client address**

> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy

**Comment by dannyob** [comment]
okay, signing this  initial application on the basis of the assertion that this is "open source" -- but could you specify the exact licenses that this data was obtained and is being distributed?

**Comment by dannyob** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacec3ji6pg4idwynojgkgvsheszxdqrbm4hzqm2rvpwiw7ymmevfylu 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 50.00TiB
#### Signer Address
> f1k6wwevxvp466ybil7y2scqlhtnrz5atjkkyvm4a
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacec3ji6pg4idwynojgkgvsheszxdqrbm4hzqm2rvpwiw7ymmevfylu

**Comment by Tom-OriginStorage** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceblzt7mch4fffxq76fbwr7amihichbtsvp3grkvorysoqgwj2o7va 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 50.00TiB
#### Signer Address
> f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceblzt7mch4fffxq76fbwr7amihichbtsvp3grkvorysoqgwj2o7va

**Comment by BigD-Exchange** [comment]
@fengyunduHJKG Hi! Great to see you have gotten approval for DataCap. If you are looking for storage providers to store these data or have any questions, please visit #bigdata-exchange on Filecoin Slack or reply here. 

We have strong demand from a diverse group of SPs, who are actively looking to onboard more data.

**Comment by fireflyHZ** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacece46lflenorjjs5pe44mu5kjpxk43tfudhsssatpv4lin2jjsni2 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1fg6jkxsr3twfnyhdlatmq36xca6sshptscds7xa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacece46lflenorjjs5pe44mu5kjpxk43tfudhsssatpv4lin2jjsni2

**Comment by ipfscn** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceb7hzm6zibriul6xwf6zrnbifmhrtqcfhes7e6i665n7z2wk6xklg 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1j4n74chme7whbz3yls4a7ixqewb6dijypqg2a3a
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceb7hzm6zibriul6xwf6zrnbifmhrtqcfhes7e6i665n7z2wk6xklg

**Comment by Tom-OriginStorage** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecqvfkj7txg5qb5wbd4av2q5vsy6opfea7dx45fqy2mtxfugoiw64 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 200.00TiB
#### Signer Address
> f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecqvfkj7txg5qb5wbd4av2q5vsy6opfea7dx45fqy2mtxfugoiw64

**Comment by kernelogic** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedkomgyu7fr5332wbvryigrbhfockaefses3dfir3uad6md62ftyq 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 200.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedkomgyu7fr5332wbvryigrbhfockaefses3dfir3uad6md62ftyq

**Comment by newwebgroup** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebutlahla37hpizt7q722v72fdtno3pnm6i6hcup64g34ni76qpgq 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebutlahla37hpizt7q722v72fdtno3pnm6i6hcup64g34ni76qpgq

**Comment by fireflyHZ** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebuewb3hqkvgahegbbq3g7rfclhqi7itxisrqo2nut3wbhb4b3hme 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1fg6jkxsr3twfnyhdlatmq36xca6sshptscds7xa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebuewb3hqkvgahegbbq3g7rfclhqi7itxisrqo2nut3wbhb4b3hme

**Comment by Tom-OriginStorage** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacec2v7nvzmgrupeditfvoz6cgw7eb2cmudptrtul52kjbqkwk2ih32 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacec2v7nvzmgrupeditfvoz6cgw7eb2cmudptrtul52kjbqkwk2ih32

**Comment by stcouldlisa** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacearagebnxolcacubpwv42q7e2tsx6gqi7z27qqc4gwrjmo4p75org 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1jvvltduw35u6inn5tr4nfualyd42bh3vjtylgci
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacearagebnxolcacubpwv42q7e2tsx6gqi7z27qqc4gwrjmo4p75org

**Comment by xiaoyuaiheshui** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceb76had65gg6s3yvn7y24xys3celyy44rn6tppxwjoeeikvx3nid2 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f122qmy25wdtt5mxd77kndiq7z5x2n3iwiuz2wdsa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceb76had65gg6s3yvn7y24xys3celyy44rn6tppxwjoeeikvx3nid2

**Comment by kernelogic** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedrxctvptfhzm2ktt557cb64qvpxlcpl64i354duelicbborkplig 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedrxctvptfhzm2ktt557cb64qvpxlcpl64i354duelicbborkplig

**Comment by Stoibers** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceded6hxc2bngnypmw4nkf3xjrikxc7j3y7fq373bfje4pe57jeypy 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1yayfsv6whu3rheviucvventj3y6t542xfpb47ei
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceded6hxc2bngnypmw4nkf3xjrikxc7j3y7fq373bfje4pe57jeypy

**Comment by Tom-OriginStorage** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceaamfntoeh2bb3gtaqgrh2lqtr4hegxdwphvmku6kxfvrcdv7cohg 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceaamfntoeh2bb3gtaqgrh2lqtr4hegxdwphvmku6kxfvrcdv7cohg

**Comment by ipfscn** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedj2h6ou64fhx54j42n4aw3sizvfvqvkpxzirogywajb72mu2l7rq 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1j4n74chme7whbz3yls4a7ixqewb6dijypqg2a3a
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedj2h6ou64fhx54j42n4aw3sizvfvqvkpxzirogywajb72mu2l7rq

**Comment by newwebgroup** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaced26qr2k344hyatulgesgjtvk3ztricz7sp4gv6f2to642zqmcy7q 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaced26qr2k344hyatulgesgjtvk3ztricz7sp4gv6f2to642zqmcy7q

**Comment by xiaoyuaiheshui** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecrz7vht7utgfugqmp32xmzb5cmxrdszxhtadefcs3kmo5guqul66 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f122qmy25wdtt5mxd77kndiq7z5x2n3iwiuz2wdsa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecrz7vht7utgfugqmp32xmzb5cmxrdszxhtadefcs3kmo5guqul66

**Comment by NDLABS-OFFICE** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceb4noamss4jf5wdknuy2yvfuxw4saalx2ehdqgnt7lspmot37vnnk 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 370.00TiB
#### Signer Address
> f1yayfsv6whu3rheviucvventj3y6t542xfpb47ei
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceb4noamss4jf5wdknuy2yvfuxw4saalx2ehdqgnt7lspmot37vnnk

**Comment by newwebgroup** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecljxlwmgzggemeneyvoapkcan4f4yoy76pmbfgyuyks724mw5opg 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 370.00TiB
#### Signer Address
> f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecljxlwmgzggemeneyvoapkcan4f4yoy76pmbfgyuyks724mw5opg

**Comment by kernelogic** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacea2w42jtadznirhkewpgn5jvnolpuypqgdmhopz5vjgkqxm6fkoca 
#### Address 
> f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy
#### Datacap Allocated
> 370.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacea2w42jtadznirhkewpgn5jvnolpuypqgdmhopz5vjgkqxm6fkoca

**Comment by filplus-checker** [comment]
## DataCap and CID Checker Report[^1]
 - Organization: `Beijing Fengyundu Technology Co.,Ltd.`
 - Client: `f1ze7kwqrzijggnis6kk7symfsis3ulaerpg5eezy`
### Storage Provider Distribution
The below table shows the distribution of storage providers that have stored data for this client.

If this is the first time a provider takes verified deal, it will be marked as `new`.

For most of the datacap application, below restrictions should apply.
 - Storage provider should not exceed 25% of total datacap.
 - Storage provider should not be storing duplicate data for more than 20%.
 - Storage provider should have published its public IP address.
 - All storage providers should be located in different regions.

✔️ Storage provider distribution looks healthy.

| Provider                                              |                           Location | Total Deals Sealed | Percentage | Unique Data | Duplicate Deals |
| :---------------------------------------------------- | ---------------------------------: | -----------------: | ---------: | ----------: | --------------: |
| [f01852325](https://filfox.info/en/address/f01852325) | Hong Kong, Central and Western, HK |         789.47 TiB |     15.42% |  789.44 TiB |           0.00% |
| [f01852023](https://filfox.info/en/address/f01852023) |                   Busan, Busan, KR |         777.00 TiB |     15.18% |  776.97 TiB |           0.00% |
| [f01851482](https://filfox.info/en/address/f01851482) |                   Busan, Busan, KR |         587.41 TiB |     11.47% |  587.41 TiB |           0.00% |
| [f01852677](https://filfox.info/en/address/f01852677) |    Morrisville, North Carolina, US |         516.03 TiB |     10.08% |  516.00 TiB |           0.01% |
| [f01852664](https://filfox.info/en/address/f01852664) |           Singapore, Singapore, SG |         478.34 TiB |      9.34% |  478.31 TiB |           0.01% |
| [f01938714](https://filfox.info/en/address/f01938714) |     Sham Shui Po, Sham Shui Po, HK |         447.94 TiB |      8.75% |  447.94 TiB |           0.00% |
| [f01938718](https://filfox.info/en/address/f01938718) |    Morrisville, North Carolina, US |         396.56 TiB |      7.75% |  396.56 TiB |           0.00% |
| [f01938721](https://filfox.info/en/address/f01938721) | Hong Kong, Central and Western, HK |         389.34 TiB |      7.60% |  389.28 TiB |           0.02% |
| [f01938665](https://filfox.info/en/address/f01938665) |     Sham Shui Po, Sham Shui Po, HK |         376.50 TiB |      7.35% |  376.50 TiB |           0.00% |
| [f01938717](https://filfox.info/en/address/f01938717) |           Singapore, Singapore, SG |         361.41 TiB |      7.06% |  361.41 TiB |           0.00% |

![Provider Distribution](https://raw.githubusercontent.com/data-preservation-programs/filplus-checker-assets/main/filecoin-project/filecoin-plus-large-datasets/issues/490/1671097244025.png)
### Deal Data Replication
The below table shows how each many unique data are replicated across storage providers.
- No more than 25% of unique data are stored with less than 4 providers.

✔️ Data replication looks healthy.

| Unique Data Size | Total Deals Made | Number of Providers | Deal Percentage |
| ---------------: | ---------------: | ------------------: | --------------: |
|        50.97 TiB |        50.97 TiB |                   1 |           1.00% |
|       143.38 TiB |       286.78 TiB |                   2 |           5.60% |
|       290.16 TiB |       870.50 TiB |                   3 |          17.00% |
|       411.03 TiB |         1.61 PiB |                   4 |          32.11% |
|       453.50 TiB |         2.21 PiB |                   5 |          44.29% |

![Replication Distribution](https://raw.githubusercontent.com/data-preservation-programs/filplus-checker-assets/main/filecoin-project/filecoin-plus-large-datasets/issues/490/1671097244666.png)
### Deal Data Shared with other Clients
The below table shows how many unique data are shared with other clients.
Usually different applications owns different data and should not resolve to the same CID.

✔️ No CID sharing has been observed.

[^1]: To manually trigger this report, add a comment with text `checker:manualTrigger`
